js/tiles.js

```diff
@@ -317,7 +317,16 @@ const door = {
             alertInfo("Saarthak Chaturvedi Drew this", "Should he become an artist?");
         },
     }),
-    black: Object.assign({}, OpenDoors, {
+	
+	sea: Object.assign({}, OpenDoors, {
+		createImages: function(){
+			this.wallTop = this.createImage("tiles/rooms/wall/seasidewall.svg");
+            this.wallRight = this.createImage("tiles/rooms/door/seasidedoor.svg");
```

Comment:
These are images with a wall and an open door. Thus I think, open doors is not what you would like to use. There should be a closed door.